### PR TITLE
Initial version of tech strategy

### PR DESCRIPTION
**Pull request by mitshubh**
I wrote up an initial version touching a few things that I could think of. I'll treat this as a living document and broaden this to include other aspects towards tenets/principles etc.

### CODE REVIEW

**Comment by brettstack** [comment]
Could you please convert this to MD so we can review in PR? :)

**Comment by mitshubh** [comment]
> Could you please convert this to MD so we can review in PR? :)

good call! done.

**Comment by sanathkr** [comment]
Great start @mitshubh! I really like the direction you are headed. The hierarchy of concepts is similar to #8 so we are in alignment.

Next step: We should get more specific with recommendations and configurations for the AWS services we care. You could even list down some CloudFormation template snippets if it helps define the recommendations. Lambda, DynamoDB, AppSync, S3, KMS, IAM are the key services IMO. Not sure if I missed any here

**Comment by mitshubh** [comment]
Thanks @sanathkr. I think you got it right in terms of the key services. Some of the others are Cloudwatch, Cognito, AWS Config and X-Ray. I think listing generic Cfn templates wouldn't be of that much help as configurations can change with environment and intention, and the basic configuration examples are already listed on the AWS Docs. Let me know if you're looking for something more here.

File: Technical_Strategy.md [hunk 1]
```diff
@@ -17,3 +17,94 @@ A lot of workflow management data in our systems is confidential and needs to be
 *Data in-transit*: All data in transit will be encrypted using TLS 1.2, or higher. This constraint may be relaxed for data movement within a secure firewall with no access to the internet (a VPC for example). 
 
 *Data at-rest*: All data at rest will be encrypted using server-side encryption. We will use customer-managed KMS keys to ensure we have complete control over key creation and management. We’ll perform envelope encryption for encrypting all data at rest.
+
+#### Initial list of AWS Services used
```
Comment (sanathkr) [inline Technical_Strategy.md:21]: This is great! Are you planning on adding more sample templates here? If not, I am okay with merging for now

Comment (mitshubh) [reply Technical_Strategy.md:21]: We can treat this as a living document, and as we encounter more services or more confidence on things like AppSync/GraphQL API we can add it to that

Comment (sanathkr) [reply Technical_Strategy.md:21]: Good idea.

**Review by sanathkr — DISMISSED** [review]
LGTM

File: Technical_Strategy.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+1.	Build a cloud-native solution leveraging AWS as much as possible. We have a good understanding of the AWS systems and services offered. Building on AWS will help us in having fast development cycles and address a lot of security/reliability/availability concerns without a lot of effort
+2.	We’ll leverage serverless technologies wherever applicable to reduce our operational burden as well as to minimize expenditure.
+
+####Development Style
+We’ll begin with having single master accounts each for test, pre-prod and production environments. All cloud resources will be managed via cloudformation or terraform to ensure repeatability and infrastructure versioning.
+
+####Access
+We have a workspace for every foundation. This workspace, and all projects underneath, should be isolated from all other workspaces in the system and should only be accessible to specific foundations. We’ll leverage Cognito to implement authentication and achieve this level of fine-grained authorization. Users in a foundation will currently have full administrative privileges for that workspace. In future, we can tighten controls further within a workspace by creating additional Cognito user pool groups.
```
Comment (brettstack) [inline Technical_Strategy.md:12]: What is a "workspace" here?

**Review by brettstack — APPROVED** [review]

**Pull request merged**